src/dotnet/commands/dotnet-pack/NuGet/ManifestSchemaUtility.cs

```diff
@@ -41,13 +41,19 @@ internal static class ManifestSchemaUtility
         /// </summary>
         internal const string SchemaVersionV6 = "http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd";
 
+        /// <summary>
+        /// Added serviceble element under metadata.
+        /// </summary>
+        internal const string SchemaVersionV7 = "http://schemas.microsoft.com/packaging/2016/06/nuspec.xsd";
```

Comment:
release-3.5.0-beta2 will have serviceable element in the SchemaVersion8 . If Joel's PackageType change is eventually going to be merged in here, should we try to be consistent and make this SchemaVersionV8? And while merging PackageType change, add that to SchemaVersionV7?